graphix/pattern.py

```diff
@@ -270,23 +269,16 @@ def standardize(self, method="direct"):
             'direct' fast standardization implemented as `standardize_direct()`
             defaults to 'direct'
         """
-        if method == "direct":
+        if method == "direct":  # faster implementation
```

Comment:
What is the purpose to maintain two methods (`direct` and `mc`)?

Reply:
Indeed we should remove `mc` if others agree. cc @thierry-martinez @masa10-f

Reply:
I see. So I will assume `mc` can be dropped when refactoring `Pattern`.

Reply:
1a759dabbb5eb224defa2a76ead7762832c652b6 another branch for deprecating `mc` method